addons/mrp/models/stock_move.py

```diff
@@ -87,7 +87,7 @@ class StockMove(models.Model):
     byproduct_id = fields.Many2one(
         'mrp.bom.byproduct', 'By-products', check_company=True,
         help="By-product line that generated the move in a manufacturing order")
-    unit_factor = fields.Float('Unit Factor', default=1)
+    unit_factor = fields.Float('Unit Factor', compute='_compute_unit_factor', store=True)
```

Comment:
do we need it stored ?

Reply:
TODO: check updating product_uom_qty in One2many